app.config.js

```diff
@@ -15,9 +18,11 @@
       "**/*"
     ],
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
     },
     "android": {
+      
+      "googleServicesFile": process.env.GOOGLE_SERVICES_JSON,
```

Comment:
Is this only needed on android ?

Reply:
Yes, as we are only doing the build on Android